export const signUp = async (req, res, next) => {
try {

}
catch (error) {
    next(error);
}}



export const signIn = async (req, res, next) => {
try {

}
catch (error) {
    next(error);
}}

export const signOut = async (req, res, next) => {
try {

}
catch (error) {
    next(error);
}}